api/spec/v1.yaml

```diff
@@ -1175,9 +1175,10 @@ components:
           description: The RFC 3339 formatted consensus time of when the most recently indexed block was produced.
           example: *iso_timestamp_2
         latest_update:
```

Comment:
Optional: The field implies an absolute timestamp to me. You might want to consider `latest_update_age[_ms]` or similar.

Reply:
I agree. Naming-wise I vote "time_since_latest_block"

Reply:
oh, too easily confused with time since chain produced a block.

Reply:
updated to `latest_update_age_ms`